src/View/HousingPanel.java

```diff
@@ -0,0 +1,133 @@
+package View;
+
+import java.awt.*;
+import java.io.File;
+import java.io.IOException;
+import javax.imageio.ImageIO;
+import javax.swing.JPanel;
+
+/**********************************************************************
+ * The HousingPanel provide abstraction for drawing houses on a
+ * property tile on the board. It handles all of the functionality of
+ * drawing or removing a house or hotel in the space where they are
+ * placed on a board square.
+ *
+ * @author David Baas
+ * @version 3/21/2018
+ *********************************************************************/
+public class HousingPanel extends JPanel {
+
+  /**An integer to track how many houses are on the tile.*/
+  private int houseCounter;
+
+  /**A static Image to track an image of a Monopoly house.*/
+  private static Image houseImg;
+
+  /**A static Image to track an image of a Monopoly hotel.*/
+  private static Image hotelImg;
+
+   /*******************************************************************
+    * The constructor creates the images and sets the hotels to zero as
+    * well as sets the background of the panel depending on what
+    * grouping the parent property belongs to.
+    * @param color the Color this HousingPanel will be set to.
+    * @throws IOException If there is a failure loading either housing
+    * image.
+    ******************************************************************/
+  public HousingPanel(Color color) throws IOException {
+    houseCounter = 0;
+
+    houseImg = ImageIO.read(new File("res/MonopolyHouse.png"));
+    hotelImg = ImageIO.read(new File("res/MonopolyHotel.png"));
+    setOpaque(true);
+    setBackground(color);
+  }
+
+  /*******************************************************************
+   * The getHouses method returns the number of houses on this panel.
+   *
+   * @return houseCounter The number houses on this panel.
+   ******************************************************************/
+  public int getHouses() {
+    return houseCounter;
+  }
+
+  /*******************************************************************
+   * The setHouses method sets the number of houses on this panel.
+   * @param houseCounter the number of houses on this panel.
+   ******************************************************************/
+  public void setHouses(int houseCounter) {
+    if(houseCounter > 5)
+      this.houseCounter = 5;
```

Comment:
Not sure about this one either, but I think you might have to add brackets here. Not for errors but for syntax.